src/EPR.RegulatorService.Frontend.UnitTests/Web/ViewModels/RegistrationSubmissions/CancelDateRegistrationSubmissionViewModelTests.cs

```diff
@@ -0,0 +1,130 @@
+namespace EPR.RegulatorService.Frontend.UnitTests.Web.ViewModels.RegistrationSubmissions
+{
+    using System;
+    using EPR.RegulatorService.Frontend.Web.ViewModels.RegistrationSubmissions;
+    using EPR.RegulatorService.Frontend.UnitTests.Web.ViewModels.Shared.Helpers;
+
+    [TestClass]
+    public class CancelDateRegistrationSubmissionViewModelTests
+    {
+        [TestMethod]
+        public void CancelDateRegistrationSubmissionViewModel_ValidFutureDate_ShouldHaveNoValidationErrors()
+        {
+            // Arrange
+            var viewModel = new CancelDateRegistrationSubmissionViewModel
+            {
+                SubmissionId = Guid.NewGuid(),
+                Day = 15,
+                Month = 12,
+                Year = 2025 // Future date
+            };
+
+            // Act
+            var validationResults = ValidationHelper.ValidateModel(viewModel);
+
+            // Assert
+            Assert.AreEqual(0, validationResults.Count, "Expected no validation errors for a valid future cancellation date.");
+            Assert.AreEqual(new DateTime(2025, 12, 15, 0, 0, 0, DateTimeKind.Unspecified), viewModel.CancellationDate);
+        }
+
+        [TestMethod]
+        public void CancelDateRegistrationSubmissionViewModel_InvalidPastDate_ShouldHaveValidationError()
+        {
+            // Arrange
+            var viewModel = new CancelDateRegistrationSubmissionViewModel
+            {
+                SubmissionId = Guid.NewGuid(),
+                Day = 15,
+                Month = 10,
+                Year = 2023 // Past date
+            };
+
+            // Act
+            var validationResults = ValidationHelper.ValidateModel(viewModel);
+
+            // Assert
+            Assert.AreEqual(1, validationResults.Count, "Expected a validation error for a past cancellation date.");
```

Comment:
To make code consistence,  would you please update your asserts with Fluent Assertions?

Reply:
Refactored